app/konnect/gateway-manager/plugins/update-custom-plugin.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Editing or deleting a custom plugin's schema
```

Comment:
This section is about editing or deleting a schema, but guidance is provided only for the editing part. Do we intend to also add something specifically about deletion? For example, schemas cannot be deleted if currently in use.

Reply:
By "in use", do you mean that there's a plugin instance configured? So something like "Before deleting a custom plugin schema, make sure to remove all configured instances of the plugin from the control plane."

Anything else you can think of? The other thing that comes to mind is what order you want to delete things in, and what happens if you just delete the control plane side (or just the data plane side). 
- my guess is data plane first then control plane, or the order doesn't matter
- if you just delete the schema from the control plane, does the plugin keep functioning on the data plane, but without the ability to be updated?
- deleting it from a data plane seems like it won't break anything - same as when you first upload a schema to konnect and have nothing on the DPs

Reply:
> By "in use", do you mean that there's a plugin instance configured? So something like "Before deleting a custom plugin schema, make sure to remove all configured instances of the plugin from the control plane."

Yes, I think that would be enough.

> Anything else you can think of? The other thing that comes to mind is what order you want to delete things in, and what happens if you just delete the control plane side (or just the data plane side).

The key point is whether the control plane has or hasn't a custom plugin configured:
- if the schema is removed in the DP while the plugin is still configured in the CP, then when a payload is sent to the DP it will complain about the schema is now unknown
- if no custom plugins exist on the CP, then both the CP and DP are free to remove the schema in whatever order

Makes sense?